cli/flox-rust-sdk/src/models/lockfile.rs

```diff
@@ -1353,6 +1404,73 @@ mod tests {
         assert_eq!(actual_params, expected_params);
     }
 
+    #[test]
+    fn make_params_unlock_if_invalidated() {
```

Comment:
Nitpick: this feels like three separate tests, and having one huge test at first glance makes it look like there's a lot of state to track throughout the test